dream_layer_backend/run_preset_tests.py

```
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+"""
+Test runner for the preset system
+Run this to verify that the preset functionality works correctly
+"""
+
+import sys
+import os
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)